traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
ldap will be going away but currently if you login as ldap you get read-only permissions on everything. so, in my opinion, find out the ID of the root tenant (select id from tenant where name = 'root') and assign that ID to the user if isLDAP. let me know if that doesn't make sense or if  you disagree.